### PR TITLE
Implement the Virtual Container Hosts section

### LINKED ISSUE #3751: Implement the Virtual Container Hosts tab of VIC object workspace
As a UI engineer I need to provide full implementation of the Virtual Container Hosts tab detailed out in the mockup.

Details:
Mockup: https://vmware.invisionapp.com/share/DA9NSB8CE#/screens/210384830

Acceptance Criteria:

Fully functional Virtual Container Hosts page as shown in the mockup

### PR DESCRIPTION
**Pull request by jooskim**
This PR adds a section to the VIC object workspace in H5 Client which shows the information on all VCHs deployed in the current VC.

image

Fixes #3751

### CODE REVIEW

**Comment by jooskim** [comment]
@jak-atx We could review together but I'm pulling you for now as a reviewer for this PR.

File: ui/vic-ui-h5c/vic-service/src/main/java/com/vmware/vic/mvc/QueryUtil.java [hunk 1]
```diff
@@ -498,4 +516,229 @@ private static PropertySpec createPropertySpec(
...
+				   .contains(params[1].toLowerCase().trim())) {
+			   return false;
+		   }
+	   }
+	   return true;
+   }
+
+   private static String getVmPropValue(ModelObject mo, String property) {
+	   if (mo instanceof VirtualContainerHostVm) {
+		   if ("name".equals(property)) {
```
Comment (sgairo) [inline ui/vic-ui-h5c/vic-service/src/main/java/com/vmware/vic/mvc/QueryUtil.java]: Would it be good to un-nest in favor of  an and ?

File: ui/vic-ui-h5c/vic-service/src/main/java/com/vmware/vic/model/VirtualContainerHostVm.java [hunk 2]
```diff
@@ -78,7 +86,12 @@ private void processExtraConfig(ArrayOfOptionValue ovs) {
 						}
 					}
 					_clientIp = sb.toString();
-					break;
+					continue;
+				}
+
+				if (EXTRACONFIG_DOCKER_PERSONALITY_ARGS_KEY.equals(key)) {
+				    _isUsingTls = ((String)ov.getValue()).indexOf("2376") > -1;
```
Comment (jooskim) [inline ui/vic-ui-h5c/vic-service/src/main/java/com/vmware/vic/model/VirtualContainerHostVm.java]: It is a fixed port used by Docker engine server from the backend. It uses either port 2375 for  no TLS, or 2376 for TLS. For your reference, see the following screenshot showing part of the VMX config for a VCH VM.

image

Comment (jooskim) [inline ui/vic-ui-h5c/vic-service/src/main/java/com/vmware/vic/model/VirtualContainerHostVm.java]: Will make an inline comment later for clarification

Comment (chengwang86) [inline ui/vic-ui-h5c/vic-service/src/main/java/com/vmware/vic/model/VirtualContainerHostVm.java]: maybe define `final String some_name = 2376` at the very beginning and make `some_name` self-explanatory.

**Pull request merged**